src/__init__.py

```diff
@@ -25,8 +25,8 @@
 )
 
 GETTING_STARTED = (
-    "To get started with upload-charm-docs, "
-    "please refer to https://github.com/canonical/upload-charm-docs#getting-started"
+    "To get started with discourse-gatekeeper, "
+    "please refer to https://github.com/canonical/discourse-gatekeeper#getting-started"
```

Comment:
Out of curiosity, will the repository be renamed as well?

Reply:
Yes, it should. @jdkandersson was saying [here](https://github.com/canonical/upload-charm-docs/pull/210#issuecomment-1710229308) that redirects with the old name would still work